File: lua/test/exists_test.lua
-- BbcWeb SDK exists test

local sdk = require("bbc-web_sdk")

describe("BbcWebSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: rb/test/exists_test.rb
# BbcWeb SDK exists test

require "minitest/autorun"
require_relative "../BbcWeb_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = BbcWebSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
